### ISSUE TITLE
super resolution results

### ISSUE THREAD

**Issue by StudyDay-deep** [tmonjo/DeepSpaCE#2]
According to your steps, the Human_Breast_Cancer_Block_A_Section_1 data was trained, but the results obtained did not seem to be particularly good. The drawn super resolution image results did not seem to be as good as the images in the paper, and the accuracy of clustering prediction seemed to be a bit low. In my running, the valid acc=0.5274, so I would like to ask where is the problem?

**Comment by tmonjo** [comment]
Hi @StudyDay-deep,

Thank you for contacting us. The prediction accuracy actually depends on a predicting gene and parameters for training.
Would you mind telling us the command that you executed?

Thanks,

**Comment by StudyDay-deep** [comment]
The parameters I used to train DeepSpaCE for super-resolution task are according to the parameters you provided, and the results are as follows:
parameters :
        --semi_option normal \
        --seed 0 \
        --threads 8 \
        --GPUs 1 \
        --cuda \
        --transfer \
        --model VGG16 \
        --batch_size 128 \
        --num_epochs 10 \
        --lr 1e-4 \
        --weight_decay 1e-4 \
        --clusteringMethod graphclust \
        --extraSize 150 \
        --quantileRGB 80 \
        --augmentation flip,crop,color,random \
        --early_stop_max 5 \
        --cross_index 0 \
        --geneSymbols ESR1,ERBB2,MKI67

Results:
geneSymbols	corR	RMSE	RMSE_MinMax
ESR1	0.178641	0.180603	0.209357
ERBB2	0.107078	0.178698	0.284158
MKI67	0.137049	0.275764	0.352937

I trained the ClusterPredction task with the same parameters as the super-resolution task, set the epoch to 100, and got the following results:
Cluster	precision	recall	f1-score	support
Cluster1	0.481203	0.603774	0.535565	106
Cluster2	0.000000	0.000000	0.000000	31
Cluster3	0.664179	0.936842	0.777293	95
Cluster4	0.625000	0.639535	0.632184	86
Cluster5	0.457447	0.728814	0.562092	59
Cluster6	0.727273	0.526316	0.610687	76
Cluster7	0.680851	0.680851	0.680851	47
Cluster8	0.692308	0.290323	0.409091	31
Cluster9	0.714286	0.303030	0.425532	33
Cluster10	0.600000	0.342857	0.436364	35
accuracy	0.590985	0.590985	0.590985	0
macro avg	0.564255	0.505234	0.506966	599
weighted avg	0.581218	0.590985	0.565197	599

**Comment by tmonjo** [comment]
Thank you for the information.

The prediction accuracy depends on datasets and parameters.
We actually used these datasets in the paper.
https://humandbs.biosciencedbc.jp/en/hum0200-v4

The parameters of the example command is just a example.
Then, the parameters were not optimized for "Human_Breast_Cancer_Block_A_Section_1" (sample data from 10x Genomics).

The parameter optimization step is needed.
I just tested the training for "Human_Breast_Cancer_Block_A_Section_1" with 50 epochs.
I obtained these results.

geneSymbols     corR    RMSE    RMSE_MinMax
ESR1    0.342061        0.165513        0.210134
ERBB2   0.267769        0.168823        0.206037
MKI67   0.231064        0.225373        0.399576

Then, I assume that the number of epoch 10 is small at least.
I also performed the ClusterPredction task but the number of cluster was 11.
I think the dataset might not completely same with yours.

Thank you again for using DeepSpaCE.

Thanks,
